cmake/install-rules.cmake

```diff
@@ -44,45 +48,49 @@ foreach (stage IN LISTS stages)
   )
 endforeach()
 
-install(
-    TARGETS jakt_runtime
-    EXPORT JaktTargets
-    RUNTIME #
-        DESTINATION "${CMAKE_INSTALL_BINDIR}"
-        COMPONENT Jakt_Runtime
-    LIBRARY #
-        DESTINATION "${CMAKE_INSTALL_LIBDIR}"
-        COMPONENT Jakt_Runtime
-        NAMELINK_COMPONENT Jakt_Development
-    ARCHIVE #
-        DESTINATION "${CMAKE_INSTALL_LIBDIR}"
-        COMPONENT Jakt_Development
-)
+# FIXME: foreach(target IN LISTS JAKT_RUNTIME_TARGETS) for this
+function(jakt_install_runtime)
+  cmake_parse_arguments(PARSE_ARGV 0 JAKT_INSTALL "" "TARGET" "")
```

Comment:
It looks like the TARGET var is required? If so, we could just make it a normal argument to the function. If you want to keep it as a single-value-argument, we should at least complain (FATAL_ERROR) if it's not set.